src/Nowruz/modules/userProfile/components/about/useAbout.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { checkVerification, requestVerification } from 'src/core/api';
+import store, { RootState } from 'src/store';
+import { currentIdentities } from 'src/store/thunks/identity.thunks';
+
+export const useAbout = () => {
+  const identityType = useSelector<RootState, 'users' | 'organizations'>(state => {
+    return state.profile.type;
+  });
+  const [openVerifyModal, setOpenVerifyModal] = useState(false);
+  const [connectUrl, setConnectUrl] = useState<string>('');
+
+  const verifyAction = async () => {
```

Comment:
is it possible to create a general utils or service for this function?
I think we call in other places too and it's hard to change it later. it it's so, it's better create a util for it.